lepotato/debimage-lepotato.yaml

```diff
@@ -0,0 +1,87 @@
+{{- $architecture := or .architecture "arm64" -}}
+{{- $suite := or .suite "sid" -}}
+{{ $type := or .type "min" }}
+{{ $image := or .image (printf "debian-lepotato-%s-%s.img" $suite $architecture) }}
+
+architecture: {{ $architecture }}
+
+actions:
+  - action: debootstrap
+    suite: {{ $suite }}
+    components:
+      - main
+    mirror: https://deb.debian.org/debian
+    variant: minbase
+
+  - action: apt
+    description: Install extra packages
+    packages: [ sudo, openssh-server, adduser, systemd-sysv, initramfs-tools, u-boot-tools, util-linux,
+                net-tools, network-manager ]
+
+  # Initramfs's post-update and kernl's postinst hooks should installed before installing kernel package
+  - action: overlay
+    source: overlays/etc
+    destination: /etc
+
+  - action: apt
+    description: Install Kernel and U-Boot packages
+    packages: [ linux-image-arm64, u-boot-amlogic ]
+
+  - action: run
+    description: Add hostname
+    chroot: true
+    command: echo debian-{{ $suite }}-{{ $architecture }} > /etc/hostname
+
+  - action: run
+    description: Add localhost
+    chroot: true
+    command: echo "127.0.1.1\tdebian-{{ $suite }}-{{ $architecture }}" >> /etc/hosts
```

Comment:
better to install `libnss-myhostname` then you don't have to mangle `/etc/hosts` as well as hostname